Dockerfile

```diff
@@ -135,6 +135,11 @@ RUN AVAILABLE_TERRAFORM_VERSIONS="1.2.9 1.3.10 1.4.6 ${DEFAULT_TERRAFORM_VERSION
 # Creating the individual distro builds using targets
 FROM alpine:${ALPINE_TAG} AS alpine
 
+EXPOSE ${ATLANTIS_PORT:-4141}
```

Comment:
Do we also have to define `ATLANTIS_PORT` as an ENV earlier in the Dockerfile?

It might be worth testing this with a unique/non-standard atlantis port

Reply:
I'll cover this test tomorrow. The "EXPOSE" does nothing actually - it merely is used for documentation purposes and is used to "show" that something happens with a specific port in the container.

I think it's only relevant in the final image as that's where the healthcheck happens.

Testing a different port is on my todo for tomorrow 👍🏼

Reply:
Works with port 4143

![image](https://github.com/runatlantis/atlantis/assets/58337159/171f539c-4a5e-42e3-88cd-725b01f7de11)

Reply:
This is tested and deployed in an environment where atlanits plan/apply workflows are also tested?

Reply:
Yep, on GCP